docs/source/60_building_apps.md

```diff
@@ -103,15 +109,16 @@ ADAM ACTIONS
 $ ./bin/adam-submit myCommand input.foo
 ```
 
-Then consider making a pull request to include the new command in ADAM!
+Then consider making a
+[pull request](https://github.com/bigdatagenomics/adam/pulls) to include the
```

Comment:
So add a section somewhere about contributing to the repo? Should this be a new doc, or does it belong in the into section?